core/pyproject.toml

```diff
@@ -1,2 +1,34 @@
-[tool.black]
+[tool.ruff]
 line-length = 160
+lint.isort.known-first-party = ["autogluon"]
+lint.isort.known-third-party = [
+    "gluonts",
+    "mxnet",
+    "networkx",
+    "numpy",
+    "pandas",
+    "psutil",
+    "pytest",
+    "scipy",
+    "sklearn",
+    "sktime",
+    "statsmodels",
+    "tqdm",
+    "Pillow",
+    "boto3",
+    "requests",
+    "timm",
+    "torch",
+    "torchvision",
+    "fairscale",
+    "scikit-image",
+    "smart_open",
+    "lightning",
+    "torchmetrics",
+    "transformers",
+    "nptyping",
+    "omegaconf",
+    "pytorch-metric-learning",
+    "nlpaug",
+    "nltk",
+]
```

Comment:
We need to duplicate the `tool.ruff.lint.isort` sections in all `pyproject.toml` files for the ruff import sorter to work correctly.